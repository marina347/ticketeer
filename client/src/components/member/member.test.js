import React from "react";
import { shallow } from "enzyme";
import Member from "./member.component";

it("Should render Member component", () => {
  expect(
    shallow(<Member name={"name"} email={"email"} picture={"picture"} />)
  ).toMatchSnapshot();
});
