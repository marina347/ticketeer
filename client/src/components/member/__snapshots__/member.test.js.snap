// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Member component 1`] = `
<div>
  <p>
    name
  </p>
</div>
`;
